content/sensu-enterprise-dashboard/2.9/rbac/overview.md

```diff
@@ -226,6 +227,20 @@ type           | String
 allowed values | any length string that only contains URL-friendly characters. _PRO TIP: we recommend using a random string generator for access tokens; e.g.: <br><code>openssl rand -base64 40 &#124;  tr -- '+=/' '-&#95;~'</code>._
 example        | {{< highlight shell >}}"accessToken": "OrIXC7ezuq0AZKoRHhf~oIl-98dX5B23hf8KudfcqJt5eTeQjDDGDQ__"{{< /highlight >}}
 
+fallback    | 
+---------------|------
+description    | Used to give an authenticated user the attributes defined in that role if that user is not found in any other defined dashboard role.
```

Comment:
So would it be true that only one role should have the fallback attribute?

Reply:
Yup, that's my understanding of the attribute. It acts as an overall `default`.